docs/backendFlow.md

```diff
@@ -0,0 +1,243 @@
+# **Backend Flow 🕸️**
+
+<br>
+
+> ## Explaining the flow using `Users` created in the project ✌️
+
+<br>
```

Comment:
Do we really need `<br>` tags? 🤔 
I guess 2 empty lines in MD should add one empty line

Reply:
Should I add the screenshots ?

Reply:
No I meant the `<br>` tag
Is it necessary?

Reply:
To give some spaces 
Else it will not look properly spaced